### PR TITLE
Feat/background rwd

### PR DESCRIPTION
**Pull request by furecool**
## Types of changes

* **Update background image & RWD system**

## Description

Set RWD-related class in tailwind.config.js, and use in PageWrapper.vue.
Import images into PageWrapper.vue as background.

## Steps to Test This Pull Request

1. Set RWD-related class in tailwind.config.js. Include extend and screens in theme.
2. In PageWrapper.vue, add  those classes in classObject() ,whitch is a method binding to the div as a class.
3. Add images to '@/static/img/background'.
4. Import images from '@/static/img/background' into PageWrapper.vue.
5. Binding those images as background style
6. In css, set max width that background-image don't display.

## Expected behavior

Add background images , and update RWD system.

## Related Issue

#207 , #208

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -13,7 +13,26 @@ module.exports = {
...
+            width: {
+                '272px': '272px',
+                '706px': '706px',
+                '1040px': '1040px',
+            },
+            padding: {
+                '24px': '24px',
+                '32px': '32px',
+                '168px': '168px',
+            },
```
Comment (flynnhou) [inline tailwind.config.js:26]: If we're already using [arbitrary value support](https://v2.tailwindcss.com/docs/just-in-time-mode#arbitrary-value-support), do we still need to specify the values in the config?
It looks redundant to me.

**Review by flynnhou — CHANGES_REQUESTED** [review]
The implementation in `PageWrapper.vue` is fundamentally not idiomatic regarding CSS and TailwindCSS.

## Problems

1. The inline [`backgroundStyle`](https://github.com/pycontw/pycontw-frontend/blob/90105f0693093a7bd8abc540518b61c95ff876ea/components/core/i18n/PageWrapper.vue#L28) made it difficult to toggle the `background-image` on different screen sizes.
2. Hence, the unideal [`!important`](https://github.com/pycontw/pycontw-frontend/blob/90105f0693093a7bd8abc540518b61c95ff876ea/components/core/i18n/PageWrapper.vue#L92) is introduced to override the inline style because of [CSS specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity).

## Suggestions

Extend tailwind config with the custom background rules:

###### tailwind.config.js

```js
module.exports = {
    ...
    theme: {
        extend: {
            backgroundImage: {
                wrapper:
                    'url("~/static/img/background/l1.svg"), url("~/static/img/background/l2.svg"), url("~/static/img/background/l3.svg"), url("~/static/img/background/r1.svg"), url("~/static/img/background/r2.svg"), url("~/static/img/background/r3.svg"), url("~/static/img/background/l1.svg"), url("~/static/img/background/l2.svg"), url("~/static/img/background/l3.svg"), url("~/static/img/background/r1.svg"), url("~/static/img/background/r2.svg"), url("~/static/img/background/r3.svg")',
            },
            backgroundPosition: {
                'wrapper-top-left':
                    '4% 100px, 4% 958px, 4% 1816px, 96% 200px, 96% 1058px, 96% 1916px, 4% 2674px, 4% 3532px, 4% 4390px, 96% 2774px, 96% 3632px, 96% 4490px',
            },
        },
    },
    ...
}
```

This way, we can utilize Tailwind's syntax sugar to deal with the background-image on different screen sizes:

###### components/core/i18n/PageWrapper.vue

```vue
<template>
    <div class="page-wrapper">
        <div :class="classObject">
            <slot></slot>
        </div>
    </div>
</template>

<script>
export default {
  // don't need `backgroundStyle()` anymore
  // only the `classObject()` updates are needed
}
</script>

<style lang="postcss" scoped>
...
.page-wrapper {
    @apply bg-none lg:bg-wrapper bg-no-repeat bg-wrapper-top-left;
}
</style>
```

**Comment by flynnhou** [comment]
@furecool Please take a look and let me know if the code review makes sense to you!

**Comment by furecool** [comment]
> @furecool Please take a look and let me know if the code review makes sense to you!

You are right, thanks!

File: components/core/i18n/PageWrapper.vue [hunk 2]
```diff
@@ -1,25 +1,30 @@
...
+            <slot></slot>
+        </div>
     </div>
 </template>
 
 <script>
 export default {
     name: 'I18nPageWrapper',
-    props: {
-        customX: { type: Boolean, default: false },
```
[the remark this answers is not in the document]
Comment (furecool) [reply components/core/i18n/PageWrapper.vue:11]: Already add this back， thanks!

[the remark this answers is not in the document]
Comment (flynnhou) [reply components/core/i18n/PageWrapper.vue:11]: @furecool Please fix the lint error QQ https://github.com/pycontw/pycontw-frontend/runs/6340634307?check_suite_focus=true#step:6:14

[the remark this answers is not in the document]
Comment (furecool) [reply components/core/i18n/PageWrapper.vue:11]: sorry, alreary fix it .

**Review by flynnhou — APPROVED** [review]
lgtm! @furecool thank you!

**Pull request merged**